### PR TITLE
Add new `no-args-paths` rule.

### PR DESCRIPTION
**Pull request by lifeart**
https://github.com/ember-template-lint/ember-template-lint/issues/841

### CODE REVIEW

File: lib/rules/index.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ module.exports = {
   'no-abstract-roles': require('./lint-no-abstract-roles'),
   'no-action-modifiers': require('./lint-no-action-modifiers'),
   'no-attrs-in-components': require('./lint-no-attrs-in-components'),
+  'no-args': require('./lint-no-args'),
```
Comment (rwjblue) [inline lib/rules/index.js]: Should this be `no-args-paths` instead? We actually _do_ allow `this.args` so saying `no-args` seems a tad bit off.

File: lib/rules/lint-no-args.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const Rule = require('./base');
+
+module.exports = class LintNoAttrs extends Rule {
```
Comment (rwjblue) [inline lib/rules/lint-no-args.js]: This class name is wrong

File: lib/rules/lint-no-args.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  visitor() {
+    function checkPathForAttrs(node) {
+      if (node.parts && node.parts[0] === 'attrs') {
```
Comment (rwjblue) [inline lib/rules/lint-no-args.js]: Seems like this should be `args` not `attrs`

**Comment by rwjblue** [comment]
@lifeart - Can you submit a follow up PR to add this to the octane config?

**Comment by lifeart** [comment]
Done https://github.com/ember-template-lint/ember-template-lint/pull/893

**Pull request merged**